inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -511,7 +516,8 @@ public void deleteProperty(KnowledgeBase kb, KBProperty aType)
         throws QueryEvaluationException
     {
         List<KBHandle> resultList = read(aKB, (conn) -> {
-            String QUERY = SPARQLQueryStore.PROPERTYLIST_QUERY;
+            String QUERY = SPARQLQueryStore
```

Comment:
`QUERY` is not a constant (`final`). All capital names should only be used for constants and enum values (which are constants).